### ISSUE TITLE
Cleaning: Insert blank line padding after closing brace (SA1513)

### ISSUE THREAD

**Issue by codecadwallader** [codecadwallader/codemaid#250]
Migrated from https://trello.com/c/t3ZG4jsB with 10 votes

```csharp
    if( condition one ) {
        line one;
        line two;
        line three;
    }
    if( condition two || condition three ) {
        line four;
        line five;
    }
```

In this case it would be preferable to place a blank line between the if statements.

Exceptions would apply to chained statements (if-else-else if, try-catch-finally) as well as nested blocks.

NOTE: This feature is dependent on a conversion to the Roslyn APIs.

> This is SA1513 btw. However there is another StyleCop rule that will not allow the blank line if the next token is another } (SA1508) so that needs to be considered when looking at this.

**Comment by ThoughtDrive** [comment]
I agree. I've been waiting for years for JetBrains to implement this feature in ReSharper. It would be great to have it available in CodeMaid.

**Comment by anshulahuja98** [comment]
Is there any update on this issue? @codecadwallader

**Comment by codecadwallader** [comment]
@anshulahuja98 This feature request is still dependent on Roslyn APIs which we have not implemented.
